db/tags.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS `Events_Tags`;
+DROP TABLE IF EXISTS `Tags`;
+
+CREATE TABLE `Tags` (
+  `Id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `TagName` varchar(64) CHARACTER SET latin1 COLLATE latin1_bin NOT NULL DEFAULT '',
```

Comment:
I would call this column just Name instead of TagName.  Simply to be consistent with our other tables.

Reply:
Good point.  I was trying to be as consistent as possible with existing code.  I'll change this to Name.